<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KNewEquityEntryDlg</class>
 <widget class="QDialog" name="KNewEquityEntryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Equity</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblEquityName">
       <property name="text">
        <string>Equity Name:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblType">
       <property name="text">
        <string>Investment Type:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KLineEdit" name="edtMarketSymbol">
       <property name="toolTip">
        <string>Trading Symbol of the stock or mutual fund, not required.</string>
       </property>
       <property name="whatsThis">
        <string>Trading Symbol of the stock or mutual fund, not required.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel1_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>1 /</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AmountEdit" name="edtFraction" native="true"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSymbol">
       <property name="text">
        <string>Market Symbol:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Smallest fraction:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KComboBox" name="cmbInvestmentType">
       <item>
        <property name="text">
         <string>Stock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mutual Fund</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bond</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="edtEquityName">
       <property name="toolTip">
        <string>Name of the company, or mutual fund.</string>
       </property>
       <property name="whatsThis">
        <string>Name of the company, or mutual fund.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>AmountEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">amountedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtEquityName</tabstop>
  <tabstop>edtMarketSymbol</tabstop>
  <tabstop>cmbInvestmentType</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KNewEquityEntryDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
